doc/Alerting/Transports.md

```diff
@@ -174,6 +174,15 @@ website and setup the transport.
 | ------ | ------- |
 | Access Token | i23f23mr23rwerw |
 
+## Browser Push
```

Comment:
Need to add docs about creating keys manually using
 `lnms tink --execute="\Minishlink\WebPush\VAPID::createVapidKeys()"`

Reply:
Hmm, `php artisan webpush:vapid`  will insert into .env automatically

Reply:
@Jellyfrog we generate keys for every install.

Reply:
And current installs?

Reply:
Ok, i  thought populateEnv only ran on newinstalls